impl/src/test/java/org/ehcache/impl/internal/store/heap/BaseOnHeapStoreTest.java

```diff
@@ -1188,9 +1188,7 @@ public void testEvictionDoneUnderEvictedKeyLockScope() throws Exception {
       try {
         key = keyExchanger.exchange("ready to roll!");
         store.put(key, "updateValue");
-      } catch (InterruptedException e) {
-        e.printStackTrace();
-      } catch (StoreAccessException e) {
+      } catch (InterruptedException | StoreAccessException e) {
```

Comment:
Since this is a test, we probably can remove the printStackTrace and just let the exception by rethrown by the test

Reply:
Letting the exception bubble up is a mess, so that's just to have info to discover issue in case they ever happen. Would rather leave it like that.